packages/adminrouter/extra/src/lib/cache.lua

```diff
@@ -59,7 +59,15 @@ end
 
 
 local function request(url, accept_404_reply, auth_token)
-    local headers = {["User-Agent"] = "Master Admin Router"}
+    -- We need to make sure that Nginx does not reuse the TCP connection here,
```

Comment:
Very interesting bug!

Reply:
I was wondering about it too, since most of our code has been moved towards not closing connections.